es-MX/Pergamino_de_Personajes/themes/pdf-theme.yml

```diff
@@ -0,0 +1,73 @@
+extends:
+  - default-sans-with-font-fallbacks
+font:
+  catalog:
+    merge: true
+    AtkinsonHyperlegibleNext:
+      normal: atkinson_hyperlegible/AtkinsonHyperlegibleNext-Regular.ttf
+      italic: atkinson_hyperlegible/AtkinsonHyperlegibleNext-RegularItalic.ttf
+      bold: atkinson_hyperlegible/AtkinsonHyperlegibleNext-Bold.ttf
+      bold_italic: atkinson_hyperlegible/AtkinsonHyperlegibleNext-BoldItalic.ttf
+    AtkinsonHyperlegibleMono:
+      normal: atkinson_hyperlegible/AtkinsonHyperlegibleMono-Regular.ttf
+      italic: atkinson_hyperlegible/AtkinsonHyperlegibleMono-RegularItalic.ttf
+      bold: atkinson_hyperlegible/AtkinsonHyperlegibleMono-Bold.ttf
+      bold_italic: atkinson_hyperlegible/AtkinsonHyperlegibleMono-BoldItalic.ttf
+    Roboto:
+      normal: roboto/roboto-v48-latin_latin-ext_math_symbols-regular.ttf
+      italic: roboto/roboto-v48-latin_latin-ext_math_symbols-italic.ttf
+      bold: roboto/roboto-v48-latin_latin-ext_math_symbols-700.ttf
+      bold_italic: roboto/roboto-v48-latin_latin-ext_math_symbols-700italic.ttf
+  fallbacks:
+    - Roboto
+    - Noto Sans
+    - Noto Emoji
+base:
+  font-size: 12
+  font-family: AtkinsonHyperlegibleNext
+page:
+  size: A4
+title-page:
+  logo:
+    align: left
+    image: image:../../../assets/images/Free5e-Logo-Gold-Ring.png[pdfwidth=20%]
+    top: 0%
+  title:
+    font-color: #b77a2b
+    font-family: Helvetica
+    font-size: 32
+    font-style: bold
+    top: 15%
+  authors:
+    content:
+      name_only: '{author}'
+    font-color: #29326a
+    font-family: Helvetica
+    font-size: 20
+footer:
+  border-style: dashed
+  height: 0.5in
+  sectlevels: 2
+  title-style: toc
+  recto:
+    left:
+      content: 'Última actualización: {revdate}'
```

Comment:
This is another Google Translate translation. In the footer it shows up like this:
<img width="285" height="80" alt="Latest update in the footer" src="https://github.com/user-attachments/assets/ff6b067c-f3f8-402a-9b49-82e213ade2b9" />